docs/changelog.md

```diff
@@ -149,6 +149,14 @@ we clean up all configurations. This could lead to unexpected new log lines. Ple
 !!! note 
     Log4j is only used for standalone server deployments and JanusGraph testing.
 
+##### Add support for Java 11
+
+All tests are now running on Java 11. We encourage everyone to test out Java 11.
+
+!!! note
+    Currently, we aren't able to release full distribution build with Java 11. 
```

Comment:
I'm not sure what this means to be honest. We still build both distribution zip archives with Java 8, right? Does the full distribution then also require Java 8 as its runtime?
Or is this just informing users that we currently cannot build the full distribution with Java 11, although we also don't build the normal distribution with Java 8?

Reply:
I added an extra line

Reply:
Where? I don't see anything added in the diff, but the note is now missing completely.